rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/BatchJobInfrastructure.java

```diff
@@ -661,16 +661,15 @@ private void deleteJob(String jobID) throws RMException {
      */
     @Override
     public String getDescription() {
-        return getBatchinJobSystemName() + "Infrastructure acquires nodes from a " + getBatchinJobSystemName() +
-               " resource manager.";
+        return getBatchinJobSystemName() + " infrastructure acquires nodes from a resource manager.";
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
     public String toString() {
-        return getDescription();
+        return getBatchinJobSystemName() + ", maxNodes: [" + this.maxNodes + "], nodeTimeOut: [" + nodeTimeOut + "]";
```

Comment:
It's a design choice, but I would rather use string.format() here. more elegant code ;)

Reply:
yes String.format can be used for more elegant code